services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/tormaystarkastelu/TormaystarkasteluLaskentaServiceImpl.kt

```diff
@@ -34,6 +34,7 @@ open class TormaystarkasteluLaskentaServiceImpl(
             // call something to create tormaystarkastelu with that luokittelu
             val laskentatulos =
                 TormaystarkasteluCalculator.calculateAllIndeksit(initTormaystarkasteluTulos(hanke), luokittelutulos)
+            laskentatulos.hankeGeometriatId = hanke.geometriat!!.id!!
```

Comment:
Do you think this could be done in initTormaystarkasteluTulos where I hid the other hanke value settings?

Reply:
Yes, it could and it should. Moved it there.